### PR TITLE
Forum Link & New Terms ID

### PR DESCRIPTION
**Pull request by rootelement**
Added a forum link from the view page as well as updating the production challenge terms UUID

### CODE REVIEW

File: src/components/ChallengeEditor/ChallengeView/index.js [hunk 1]
```diff
@@ -163,6 +163,15 @@ const ChallengeView = ({
                 readOnly
               />
             )}
+            <div>
+              { challenge.discussions && challenge.discussions.map(d => (
+                <div className={cn(styles.row, styles.topRow)}>
```
Comment (maxceem) [inline src/components/ChallengeEditor/ChallengeView/index.js:168]: Each rendered element in the list should have a unique `key` prop (for example `d.id` can be used):

image

**Review by maxceem — COMMENTED** [review]
@rootelement works good:

image

Just one code improvement can be done above to avoid warning in the browser console.

File: src/components/ChallengeEditor/ChallengeView/index.js (hunk 1 above)
Comment (rootelement) [reply src/components/ChallengeEditor/ChallengeView/index.js]: Fixed! Thanks!

**Review by vikasrohit — APPROVED** [review]
LGTM

**Pull request merged**